wifinina/tcp.go

```diff
@@ -12,8 +12,33 @@ const (
 	ReadBufferSize = 128
 )
 
-func (d *Device) NewDriver() net.DeviceDriver {
-	return &Driver{dev: d, sock: NoSocketAvail}
+// Configure resets wifinina device, creates network driver and tells network stack to use it.
+// Wifi connection can either be handled esplicitly in user code or transparently by using WithAccessPoint.
+func (d *Device) Configure() *Driver {
+	d.Reset()
+	drv := &Driver{dev: d, sock: NoSocketAvail}
+	net.UseDriver(drv)
+	return drv
+}
```

Comment:
How about connecting to AccessPoint by passing the argument `Config{}` to this `Configure()`?

Reply:
I've kept API unchanged intentionally.
Since many examples use `Configure` w/o params and all user code out there in the wild too.
Introduce `ConfigureWithAccessPoint` to be backwards compatible?
Settled on fluent-style (`WithAccessPoint`) to satisfy both needs. Also future-proof if we ever add more "extra" functionality.

Reply:
From a compatibility point of view, it would certainly be better to add WithAccessPoint.
Do you have any opinions? @deadprogram @bgould

Reply:
See my previous comment.